lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -9367,7 +9370,12 @@ namespace Js
 
             Var thisVar = RootToThisObject(object, scriptContext);
 
-            RecyclableObject* marshalledFunction = RecyclableObject::FromVar(CrossSite::MarshalVar(requestContext, function));
+            RecyclableObject* marshalledFunction = function;
+            if (scriptContext != requestContext)
```

Comment:
can you make an overload of CrossSite::MarshalVar() to take both scriptContext as input parameter, to make the caller code cleaner

Reply:
@leirocks good point. I was thinking about it, let me update the PR.

Reply:
+1, I like that convention.